src/outagesim/backup_reliability.jl

```diff
@@ -240,60 +466,69 @@ If ``marginal_survival`` = true then result is chance of surviving in given outa
 if ``marginal_survival`` = false then result is chance of surviving up to and including given outage hour.
```

Comment:
Instead of calling this function (and the with battery version) twice, with marginal_survival true and false, just make this function calculate the marginal version and calculate the other from that matrix. I think there's a function that does exactly that, returns a matrix where each element is the product of input matrix elements up to that index, but don't remember what it's called. Doing that should actually make an impact on speed because this is the part that takes longest I'm assuming.

Reply:
@sericson0 this is not yet addressed

Reply:
Was talking with Bill about this today that the naming of Marginal versus Cumulative is not ideal. The cumulative means that the system survives up to and including the outage while the marginal means the system can survive in the given outage hour. Unfortunately the cumulative survivability is not the product of the marginal survivals in each hour. I definitely agree that it is inefficient to run the code twice if there were a way to calculate in 1 go, just not sure how to do it.